client/app.html

```diff
@@ -4,7 +4,10 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0, user-scalable=yes">
     <link rel="stylesheet" href="../third_party/codemirror-5.19.0/lib/codemirror.css">
+    <link rel="stylesheet" href="../third_party/jquery-ui-1.12.1.custom/jquery-ui.min.css">
```

Comment:
We can't be using a custom build here. Can we just use standard jquery ui instead?

Rationale: we may need to upgrade the libraries in the future, and in doing so, would need to know exactly what to include. So, probably better to just use the default vanilla jquery UI.